pkgs/applications/science/biology/whisper/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, fetchurl }:
+
+stdenv.mkDerivation {
+  pname = "whisper";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    owner = "refresh-bio";
+    repo = "Whisper";
+    rev = "v2.0";
```

```suggestion
    rev = "v${version}";
```

https://nixos.wiki/wiki/Nix_Expression_Language#rec_statement